src/components/MetricsOptions/MetricsOptionsBar.tsx

```diff
@@ -96,6 +97,7 @@ export class MetricsOptionsBar extends React.Component<Props, MetricsOptionsStat
   };
 
   onDurationChanged = (key: number) => {
+    sessionStorage.setItem('appDuration', JSON.stringify(key));
```

Comment:
Same here, I wonder converting the object to JSON is needed.

Reply:
It's not necessary, only convert to string